ironfish/jest.setup.env.js

```diff
@@ -26,8 +26,8 @@ jest.mock('node-datachannel', () => {
 })
 
 beforeAll(() => {
-  // This causes the WASM to be initialized, which is 1 time 2 second cost for each test suite
-  if (process.env.TEST_INIT_WASM) {
+  // This causes Sapling to be initialized, which is 1 time 2 second cost for each test suite
```

Comment:
Nit: Is this supposed to be "**a** 1 time 2 second cost..." (alternatively, "a one-time two-second cost...")?